### ISSUE TITLE
error parsing query: found s, expected ;

### ISSUE THREAD

**Issue by mabushey** [popstas/transmission-cli#40]
michael@corwin # /usr/local/bin/transmission-cli

Transmission CLI version v0.7.0 build @git-commit@

Usage:
  command [options] [arguments]
...

 michael@corwin# /usr/local/bin/transmission-cli stats-send --transmission-host=localhost
[critical] error parsing query: found s, expected ; at line 1, char 93

Version v0.6.0 did the same thing.

**Comment by popstas** [comment]
What influxdb version you have? Please show output of ´/usr/local/bin/transmission-cli stats-send --transmission-host=localhost -vvv´

**Comment by mabushey** [comment]
╰─➤  /usr/bin/influx 
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version 1.2.2
InfluxDB shell 0.10.0
...

╰─➤  /usr/local/bin/transmission-cli stats-send --transmission-host=localhost -vvv
[debug] Connect Transmission using: corwin:default@localhost:9091
[info] [2017-04-11 23:56:35] command: stats-send --transmission-host=localhost -vvv
[debug] Request sent to the Transmission RPC API.
[info] Invalid Transmission session ID. A new ID has been generated.
[debug] Request sent to the Transmission RPC API.
[debug] Connect InfluxDB using: corwin:[mypass]@localhost:8086
[critical] error parsing query: found s, expected ; at line 1, char 93

**Comment by popstas** [comment]
Hm, I have
```
$ influx
Connected to http://localhost:8086 version 1.2.1
InfluxDB shell version: 1.2.2
```
stats-send works. 

Seems you have torrent with ' in names, see https://community.smartthings.com/t/smartthings-data-visualisation-using-influxdb-and-grafana/44946/27

Will check and fix at weekend. Thank you for report!

**Comment by popstas** [comment]
Confirmed, fixed in v0.7.1, please check.

**Comment by mabushey** [comment]
Thank you for fixing this. :)
